cluster/gce/gci/configure-helper-common.sh

```diff
@@ -1053,7 +1074,11 @@ function create-master-kubelet-auth {
   # set in the environment.
   if [[ -n "${KUBELET_APISERVER:-}" && -n "${KUBELET_CERT:-}" && -n "${KUBELET_KEY:-}" ]]; then
     REGISTER_MASTER_KUBELET="true"
-    create-kubelet-kubeconfig ${KUBELET_APISERVER}
+    if [[ "${ENABLE_APISERVER_INSECURE_PORT:-false}" == "true" ]]; then
+      create-kubelet-kubeconfig ${KUBELET_APISERVER} "8080" "http"
+    else
+      create-kubelet-kubeconfig ${KUBELET_APISERVER}
+    fi
```

Comment:
ENABLE_APISERVER_INSECURE_PORT not only used for tenant/resource partition. if use this as condition, may impact regular cluster logic.

Reply:
how?

Reply:
as we talked in the meeting, if this is only for POC, that's ok.